Meta/CMake/flatpak/org.ladybird.Ladybird.desktop

```diff
@@ -0,0 +1,32 @@
+[Desktop Entry]
+Name=Ladybird
+GenericName=Web Browser
+GenericName[eo]=Retumilo
+GenericName[fr]=Navigateur Web
+GenericName[ru]=Веб-браузер
+GenericName[sv]=Webbläsare
+Exec=Ladybird --force-new-process %U
+Icon=org.ladybird.Ladybird
+DBusActivatable=false
+Type=Application
+Categories=Network;WebBrowser;Qt;KDE;
+StartupNotify=false
+MimeType=text/html;application/xhtml+xml;x-scheme-handler/http;x-scheme-handler/https;
+Actions=new-window;new-incognito-window;
+X-Purism-FormFactor=Workstation;Mobile;
+
+[Desktop Action new-window]
+Name=New Window
+Name[eo]=Nova fenestro
+Name[fr]=Nouvelle Fenêtre
+Name[ru]=Новое окно
+Name[sv]=Nytt fönster
+Exec=Ladybird --new-window
+
+[Desktop Action new-incognito-window]
+Name=New Incognito Window
+Name[eo]=Nova inkognita fenestro
+Name[fr]=Nouvelle Fenêtre de Navigation Privée
+Name[ru]=Новое окно инкогнито
+Name[sv]=Nytt privat fönster
+Exec=ladybird --no-such-option
```

Comment:
update this. Ladybird --no-such-option. Or perhaps just delete. Is there a way to add a comment to these things to say "when we add this feature, add an entry"?